code/build/template/.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,43 @@
+{
+  "configurations": [
+    {
+      "name": "Linux",
+      "includePath": [
+        "${workspaceFolder}/include/**",
+        "${env:ORX}/include"
+      ],
+      "defines": [],
+      "compilerPath": "/usr/bin/gcc",

```

Comment:
Same comment as with the first file.

Reply:
Also in https://github.com/orx/orx/pull/104/commits/3b8d6d54c33fe477230ab62464e69f4a46fc1112.